### DISCUSSION TITLE
Sakura原主题的github卡片问题

### DISCUSSION THREAD

**Discussion by 503611908** [mirai-mamori/Sakurairo#308] [Q&A]
在function.php的sakura_scripts()和html_tag_parser($content)中修改后，github cards短代码无法正常显示，之前的版本是可以的，或者能不能在功能里加一个呢？

- PHP 版本：8.0
- 数据库类型 / 版本：MySQL5.7
- WordPress 版本：5.8
- 主题版本：2.2.0.1
- 浏览器：EDGE

**Comment by beixibaobao** [comment]
帮你问了一下，这个区块删除很久了（是被删除的不是BUG）
